README.md

```diff
@@ -97,8 +97,19 @@ This list contains projects using OpenStreetMap data for creative purposes, as w
 
 * [Baremaps](https://www.baremaps.com/) - Open source pipeline for producing Mapbox vector tiles from OpenStreetMap with Postgis and Java. ([Source Code](https://github.com/baremaps/baremaps))
 
+## Changeset Review & Viewer Tools
+
+### Web Services
+
+* [osmcha](https://osmcha.mapbox.com) - Detector for suspicious changesets. ([Source Code](https://github.com/mapbox/osmcha-frontend))
+* [achavi](https://overpass-api.de/achavi/) - Augmented change viewer. ([Source Code](https://github.com/nrenner/achavi/))
+
 <!--Offline, may be because of russia-ukraine-war - * [whodidit](http://zverik.osm.rambler.ru/whodidit/) - Changesets analyzer.-->
 
+### Selfhosted
```

Comment:
In my opinion this should be a part of "Web Services", as almost all of the web services can be self-hosted and adding a separate category will just complicate things

Reply:
The difference is that the 'web service' ones provide a hosted instance and can be used by anyone without any extra work while the 'selfhosted' ones do not and need to be set up on your own to be usable (which is a major hurdle that excludes 95% of users)

Reply:
I mean for the project you've listed under Selfhosted there would be a hosted instance, but it seems to be defunct. (expandable.dk seems to not have any A, AAAA or MX DNS records)

I still think a "Selfhosted" category would be irritating, as that would imply the others would not be self-hostable, but I agree osm-analytic-tracker does not really fit with the other web-services.

Maybe we should just leave osm-analytic-tracker out for now until there is a working instance again?

Reply:
Or we could add a note like with a footnote explaining the situation as follows:

(⚠️ Needs to be self-hosted[^no-running-instances])

[^no-running-instances]: Some projects have no running instances and need to be deployed to a custom server to be used. Please refer to the projects website to learn more about setting up an instance.

Reply:
Adding some sort of note would be a good idea. Not adding self-host-only tools kind of defeats the purpose of this list.

Reply:
I agree not adding those tools would not be helpful.

I think the note with a footnote explaining further is the best option, so if you'd add that I'll merge this! 😃

Reply:
I added the footnote and a bunch more tools